### ISSUE TITLE
Complete writing comprehensive automation tests

### ISSUE THREAD

**Issue by karkarl** [microsoft/WinUI-Gallery#1340]
```[tasklist]
### Tasks
- [x] Accessibility test passes for each page
- [x] Navigate to each page and scroll to the end (to catch any crash style regression)
- [ ] Check validity of each link
```

**Comment by citelao** [comment]
I'd be happy to consult on this (I'm an accessibility champ for DASH); let me know!

**Comment by chingucoding** [comment]
I'm curios if the validity link checking should be done as an automation test or if this should be a build step that just checks all links we added during compilation.
